stdlib/REPL/src/emoji_symbols.jl

```diff
@@ -76,35 +96,38 @@ const emoji_symbols = Dict(
     "\\:apple:" => "🍎",
     "\\:aquarius:" => "♒",
     "\\:aries:" => "♈",
-    "\\:arrow_backward:" => "◀",
+    "\\:arrow_backward:" => "◀️",
     "\\:arrow_double_down:" => "⏬",
     "\\:arrow_double_up:" => "⏫",
-    "\\:arrow_down:" => "⬇",
+    "\\:arrow_down:" => "⬇️",
     "\\:arrow_down_small:" => "🔽",
-    "\\:arrow_forward:" => "▶",
-    "\\:arrow_heading_down:" => "⤵",
-    "\\:arrow_heading_up:" => "⤴",
-    "\\:arrow_left:" => "⬅",
-    "\\:arrow_lower_left:" => "↙",
-    "\\:arrow_lower_right:" => "↘",
-    "\\:arrow_right:" => "➡",
-    "\\:arrow_right_hook:" => "↪",
-    "\\:arrow_up:" => "⬆",
-    "\\:arrow_up_down:" => "↕",
+    "\\:arrow_forward:" => "▶️",
```

Comment:
Are we sure that we want the emoji rendering variant of these? I feel like for source code, the text rendering might be better.